nncf/common/graph/graph.py

```diff
@@ -525,10 +534,11 @@ def _get_graph_for_visualization(self) -> nx.DiGraph:
         ignored/target scopes.
         """
         out_graph = nx.DiGraph()
-        for node_name, node in self._nx_graph.nodes.items():
+        for node in self.get_all_nodes():
             attrs_node = {}
-            attrs_node['label'] = str(node[NNCFGraph.ID_NODE_ATTR]) + ' ' + str(node[NNCFGraph.NODE_NAME_ATTR])
-            out_graph.add_node(node_name, **attrs_node)
+            attrs_node['label'] = str(node.node_id) + ' ' + str(node.node_name)
```

Comment:
I think this should work. This suggestion is at your discretion.
```suggestion
            attrs_node['label'] = f'{node.node_id} {node.node_name}'
```

Reply:
Done